arches/app/tasks.py

```diff
@@ -78,22 +79,30 @@ def export_search_results(self, userid, request_values, format, report_link):
 
     search_history_obj = models.SearchExportHistory.objects.get(pk=exportid)
 
+    expiration_date = datetime.now() + timedelta(seconds=settings.CELERY_SEARCH_EXPORT_EXPIRES)
+    formatted_expiration_date = expiration_date.strftime("%A, %d %B %Y")
+
+    user = ""
+
+    if _user.first_name != "":
+        user = _user.first_name
+    else:
+        user = _user.username
+
+    context = return_message_context(
+        _(f"Hello,\nYour request to download a set of search results is now ready. You have until {formatted_expiration_date} to access this download, after which time it'll be deleted."),
+        _("Thank you"),
+        email,
+        {"link":str(exportid),"button_text":_("Download Now"),"name":export_name,"email_link":str(settings.ARCHES_NAMESPACE_FOR_DATA_EXPORT).rstrip("/") + "/files/" + str(search_history_obj.downloadfile),"username":user},
+        )
+
     return {
         "taskid": self.request.id,
         "msg": _(
-            "Your search {} is ready for download. You have 24 hours to access this file, after which we'll automatically remove it."
-        ).format(export_name),
-        "notiftype_name": "Search Export Download Ready",
-        "context": dict(
-            greeting=_("Hello,\nYour request to download a set of search results is now ready."),
-            link=str(exportid),
-            button_text=_("Download Now"),
-            closing=_("Thank you"),
-            email=email,
-            name=export_name,
-            email_link=str(settings.ARCHES_NAMESPACE_FOR_DATA_EXPORT).rstrip("/") + "/files/" + str(search_history_obj.downloadfile),
+            f"Your search '{export_name}' is ready for download. You have until {formatted_expiration_date} to access this file, after which we'll automatically remove it."
```

Comment:
This should be wrapped in a translating function `_(text)`. Echoing the f-string comment above as well

Reply:
Amended as per request